tests/integration/test_processor.py

```diff
@@ -10,6 +10,15 @@
 )
 
 
+def test_create_token():
+    client = create_client()
+    # Just test the failure case - behavior here depends on the API keys used
+    with pytest.raises(InvalidRequestError) as e:
+        client.Processor.stripeBankAccountTokenCreate(
```

Comment:
⚠️ i think you meant to test the new method here right? looks like it's testing the stripe token create method.